--- a/keydb.conf
+++ b/keydb.conf
@@ -155,7 +155,7 @@ supervised no
 #
 # Creating a pid file is best effort: if KeyDB is not able to create it
 # nothing bad happens, the server will start and run normally.
-pidfile /var/run/keydb_6379.pid
+pidfile /run/keydb/keydb.pid
 
 # Specify the server verbosity level.
 # This can be one of:
@@ -260,7 +260,7 @@ dbfilename dump.rdb
 # The Append Only File will also be created inside this directory.
 #
 # Note that you must specify a directory here, not a file name.
-dir ./
+dir /var/lib/keydb
 
 ################################# REPLICATION #################################
 
@@ -681,7 +681,7 @@ replica-priority 100
 # limit for maxmemory so that there is some free RAM on the system for replica
 # output buffers (but this is not needed if the policy is 'noeviction').
 #
-# maxmemory <bytes>
+maxmemory 64MB
 
 # MAXMEMORY POLICY: how KeyDB will select what to remove when maxmemory
 # is reached. You can select among five behaviors:
